templates/web/smidsy/reports/index.html

```diff
@@ -1,55 +1,142 @@
-[% INCLUDE 'header.html', title = loc('Statistics'),  bodyclass => 'fullwidthpage' %]
+[% USE date -%]
+[% extra_js = [
+    version('/vendor/chart.min.js'),
+    version('/vendor/accessible-autocomplete.min.js'),
+    version('/js/dashboard.js')
+] -%]
+[% INCLUDE 'header.html', title = loc('Statistics'),  bodyclass => 'dashboard fullwidthpage' %]
 
-<h1>[% loc('Statistics') %]</h1>
-
-<div class="intro">
-<p>
-This is a summary of cycle-related incidents reported on this site. Since the data relies on user-submitted reports, you shouldn’t use it to compare the safety of different areas.
-</p>
+<div class="dashboard-header">
+    [% IF area_body_name %]
+    <h1>[% tprintf(loc('Statistics for %s'), area_body_name) %]</h1>
+    [% ELSE %]
+    <h1>[% loc('Statistics') %]</h1>
+    [% END %]
+    <p>
+        Cyclists all over the UK contribute to Collideoscope’s database of cycling incidents and near misses. You can use the filters on this page to explore the data for the whole UK, a specific area, or a particular time range. And you can download the data currently on view as a CSV, for your own use.
+    </p>
+    <p>
+        Just bear in mind, Collideoscope relies on user-submitted reports, so you shouldn’t use it to compare the absolute safety of different areas.
+    </p>
 </div>
 
-<form method="get" class="statistics-filter">
-  Show:
-  <label><input type="radio" name="type" value="all" 
-      [% IF report_filter == 'all' %]checked[% END %]> 
-      All bodies</label>
-  <label><input type="radio" name="type" value="london"
-      [% IF report_filter == 'london' %]checked[% END %]> 
-      London boroughs</label>
-  <label><input type="radio" name="type" value="city"
-      [% IF report_filter == 'city' %]checked[% END %]> 
-      Cities (unitary authorities)</label>
-  <label><input type="radio" name="type" value="dc"
-      [% IF report_filter == 'dc' %]checked[% END %]> 
-      Districts and counties</label>
+<form method="GET">
+    <div class="filters">
+        <p>
+            [% IF area_body %]
+            <input type="hidden" name="area_body" value="[% area_body %]">
```

Comment:
It doesn't happen because area_body is used as an integer first, but note this is technically just relaying user inputted data, so should be escaped.